core/bin/dust_api.rs

```diff
@@ -636,17 +636,6 @@ async fn run_helper(
     }
 
     if payload.inputs.is_some() {
-        if payload.run_type != run::RunType::Deploy {
```

Comment:
How do you feel about removing that ?
Execute tab is doing a run with `run_type=local`, but passing an inputs array instead of specifying a dataset

Reply:
I think this is fine. Maybe we need a new run type but we can think about that later